README.md

```diff
@@ -111,7 +111,7 @@ Please note that parameter values set on Spark configuration will have higher
 priority.
 
 For an example walkthrough please check
-[docs/example-walkthrough](docs/example-walkthrough.md).
+[doc/example-walkthrough](doc/example-walkthrough.md).
```

Comment:
Could you also add a link to the changelog file in README?

Reply:
Thanks! Added it.